api/v1beta1/azuremanagedcontrolplane_webhook.go

```diff
@@ -76,10 +78,20 @@ func (mw *azureManagedControlPlaneWebhook) Default(ctx context.Context, obj runt
 		networkPlugin := "azure"
 		m.Spec.NetworkPlugin = &networkPlugin
 	}
+
 	if m.Spec.LoadBalancerSKU == nil {
-		loadBalancerSKU := "Standard"
+		loadBalancerSKU := "standard"
```

Comment:
Seems like we have an opportunity to make a const out of these four LB strings.

Reply:
Fixed by deleting this altogether :)